### PR TITLE
Beyondevil/cleanup

### PR DESCRIPTION
**Pull request by BeyondEvil**

### CODE REVIEW

File: src/pytest_html/scripts/datamanager.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const { getCollapsedCategory } = require('./storage.js')
 
 class DataManager {
     setManager(data) {
-        const collapsedCategories = getCollapsedCategory()
+        const collapsedCategories = [...getCollapsedCategory(), 'passed']
```
Comment (pv-viktor) [inline src/pytest_html/scripts/datamanager.js:5]: snyggt!

File: src/pytest_html/scripts/main.js [hunk 2]
```diff
@@ -91,8 +91,15 @@ const renderDerived = (tests, collectedItems, isFinished) => {
...
         const testWord = numberOfTests > 1 ? 'tests' : 'test'
-        const innerText = `${numberOfTests} ${testWord} ran in ${formattedAccTime} seconds.`
-        document.querySelector('.run-count').innerText = innerText
+
+        let durationText
+        if (formattedAccTime.hasOwnProperty('ms')) {
+          durationText = formattedAccTime.ms
+        } else {
+          durationText = formattedAccTime.seconds
+        }
```
Comment (pv-viktor) [inline src/pytest_html/scripts/main.js]:
```suggestion
        const durationText = formattedAccTime.ms ?? formattedAccTime.seconds
```

**Review by pv-viktor — APPROVED** [review]
Looking good!

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+services:
+  chrome:
+    image: selenium/standalone-chrome:latest
+    container_name: chrome
+    shm_size: '2gb'
+    ports:
+      - "4444:4444"
+      - "7900:7900"
+    volumes:
+      - "${PWD}:/reports${PWD}:ro"
```
Comment (RonnyPfannschmidt) [inline docker-compose.yml]: why `ro` instead of `Z`

Comment (BeyondEvil) [reply docker-compose.yml]: Just experimenting, tried without  `ro` as well.

Not sure what `Z` does.

Comment (RonnyPfannschmidt) [reply docker-compose.yml]: z ensures user id mapping and selinux relabeling

aka allow to read user owned files

Comment (RonnyPfannschmidt) [reply docker-compose.yml]: it has to be capital z tho

Comment (BeyondEvil) [reply docker-compose.yml]: Oh, interesting! Maybe that's the key to solving this!

Will give it a try ASAP, thanks! 🙏

Comment (RonnyPfannschmidt) [reply docker-compose.yml]: we jsut figured that the root casue of the permissions is 
https://github.com/pytest-dev/pytest/blob/main/src/_pytest/tmpdir.py#L154-L159

--basetmp has to go :face_exhaling:

Comment (RonnyPfannschmidt) [reply docker-compose.yml]: ouch https://github.com/pytest-dev/pytest/pull/8516

File: src/pytest_html/scripts/datamanager.js (hunk 1 above)
Comment (BeyondEvil) [reply src/pytest_html/scripts/datamanager.js:5]: Tack! <3

File: src/pytest_html/scripts/main.js (hunk 2 above)
Comment (BeyondEvil) [reply src/pytest_html/scripts/main.js]: I went with the ternary option, the `??` is super nice but I feel it's too much magic for your average pythonista to handle. 😅  @drRedflint

**Pull request merged**